UsingSeal.md

```diff
@@ -254,49 +262,147 @@ Start by generating a master seed for the key server.
 Use the `seal-cli` tool as `cargo run --bin seal-cli gen-seed`.
 This command outputs the secret master seed which should be stored securely.
 
+```shell
+cargo run --bin seal-cli gen-seed
+Seed: $MASTER_SEED
+```
+
 Next, create a configuration file in .yaml format following the instructions in the [example config](crates/key-server/key-server-config.yaml) 
 and with the following properties:
 - Set the mode to `Permissioned`.
 - Initialize with an empty client configs (clients can be added later).
 
+```yaml
+  server_mode: !Permissioned
+    client_configs:
+```
+
 Set the environment variable `MASTER_SEED` to the master secret seed generated by the `seal-cli` tool,
 and the environment variable `CONFIG_PATH` pointing to a .yaml configuration file.
 Run the server using `cargo run --bin key-server`.
 It should abort after printing a list of unassigned derived public keys (search for logs with the text `Unassigned derived public key`).
 
+```shell
+MASTER_SEED=$MASTER_SEED CONFIG_PATH=crates/key-server/key-server-config.yaml cargo run --bin key-server 
+```
+
 Each supported client must have a registered on-chain key server object to enable discovery and policy validation.
 
+```
+2025-06-15T02:02:56.303459Z  INFO key_server: Unassigned derived public key with index 0: "$PUBKEY_0"
+2025-06-15T02:02:56.303957Z  INFO key_server: Unassigned derived public key with index 1: "$PUBKEY_1"
+2025-06-15T02:02:56.304418Z  INFO key_server: Unassigned derived public key with index 2: "$PUBKEY_2"
+```
+
 Register the first client:
-- Use the first unassigned derived public key (with derivation index 0).
-- Register a new key server on-chain by calling the `create_and_transfer_v1` function from the `seal::key_server` module (see Open Mode above for usage).
-- In your key server configuration file:
-  - Add a new client entry.
+
+- Register a new key server on-chain by calling the `create_and_transfer_v1` function from the `seal::key_server` module with the first unassigned derived public key (with derivation index 0) $PUBKEY_0.
+
+```shell
+sui client call --function create_and_transfer_v1 --module key_server --package 0x62c79dfeb0a2ca8c308a56bde530ccf3846535e1623949d45c90d23128afff52 --args $YOUR_SERVER_NAME https://your-url-here 0 $PUBKEY_0 --gas-budget 10000000
+# outputs $KEY_SERVER_OBJECT_ID_0
+```
+
+
+- Add an entry in config file:
   - Set `client_master_key` to type `Derived` with `derivation_index` as 0. 
+  - Set `$KEY_SERVER_OBJECT_ID_0` from the output above. 
```

Comment:
it's not an env variable, the config should have the actual value

Reply:
i use a <> instead now